src/wallet/rpcdump.cpp

```diff
@@ -819,9 +819,97 @@ UniValue dumpwallet(const JSONRPCRequest& request)
     return reply;
 }
 
+struct ImportData
+{
+    // Input data
+    std::unique_ptr<CScript> redeemscript; //! Provided redeemScript; will be moved to `import_scripts` if relevant.
+    std::unique_ptr<CScript> witnessscript; //! Provided witnessScript; will be moved to `import_scripts` if relevant.
+
+    // Output data
+    std::set<CScript> import_scripts;
+    std::set<CKeyID> used_keys; // Import these private keys if available
+    std::set<CKeyID> require_keys; // These public keys are required for solvability
```

Comment:
I think it'd be good to mention that `require_keys` is a subset of `used_keys`. Could also choose a more normalized representation like `std::map<CKeyID, bool /* required */> used_keys`.

Reply:
Done.